dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/BeanDeploymentArchiveImpl.java

```diff
@@ -369,6 +377,9 @@ private void initializeJEEComponentClasses(Set<String> allClassNames) throws CDI
         for (EjbDescriptor<?> ejb : ejbs) {
             if (ejb.isMessageDriven()) {
                 classes.add(ejb.getBeanClass());
+                if (ejb.getBeanClass() == null) { 
```

Comment:
For this case (and the one for managed beans) do we want to add the class to `classes` if it's `null`?

Reply:
As I said in slack. I think for now we stick to the existing behaviour, but log it.